### PR TITLE
checkNotTranslation error message fix

### PR DESCRIPTION
**Pull request by msladek**
https://synjira.atlassian.net/browse/CELDEV-408

### CODE REVIEW

File: src/main/java/com/celements/model/access/DefaultModelAccessFacade.java [hunk 1]
```diff
@@ -359,9 +358,10 @@ public BaseObject getXObject(XWikiDocument doc, DocumentReference classRef, Stri
   }
 
   private void checkNotTranslation(XWikiDocument doc) {
-    String msg = "Trying to access XObjects on translation '{0}' of doc '{1}";
-    checkState(!isTranslation(doc), MessageFormat.format(msg, doc.getLanguage(),
-        doc.getDocumentReference()));
+    if (isTranslation(doc)) {
```
Comment (fpichler) [inline src/main/java/com/celements/model/access/DefaultModelAccessFacade.java]: keep using MessageFormat.format but fix the wrong usage. I expect that the single quotes cause the problem. Extract the Error-Message to a constant and check in a simple unit test that the replacement with MessageFormat is working.

**Review by fpichler — CHANGES_REQUESTED** [review]

File: src/main/java/com/celements/model/access/DefaultModelAccessFacade.java [hunk 2]
```diff
@@ -358,8 +358,8 @@ public BaseObject getXObject(XWikiDocument doc, DocumentReference classRef, Stri
     return Collections.unmodifiableMap(ret);
   }
 
-  private void checkNotTranslation(XWikiDocument doc) {
-    String msg = "Trying to access XObjects on translation '{0}' of doc '{1}";
+  void checkNotTranslation(XWikiDocument doc) {
+    String msg = "Trying to access XObjects on translation ''{0}'' of doc ''{1}''";
```
Comment (fpichler) [inline src/main/java/com/celements/model/access/DefaultModelAccessFacade.java]: please extract to constant.

**Review by fpichler — CHANGES_REQUESTED** [review]

File: src/main/java/com/celements/model/access/DefaultModelAccessFacade.java (hunk 2 above)
Comment (msladek) [reply src/main/java/com/celements/model/access/DefaultModelAccessFacade.java]: what for? it is only used in this exact method.

Comment (fpichler) [reply src/main/java/com/celements/model/access/DefaultModelAccessFacade.java]: for better readability. You may inline instead, but do not assign a constant final string to a local variable.

**Review by fpichler — APPROVED** [review]

**Pull request merged**